packages/utils/src/DOMUtils.ts

```diff
@@ -28,4 +28,29 @@ export function identityExtractHTMLElement<T>(
   return maybeHTMLElement instanceof HTMLElement ? maybeHTMLElement : null;
 }
 
-export default { getClosestByClassName, identityExtractHTMLElement };
+/**
+ * Synchronize CSS animations with a given name to the same start time.
+ * @param animationName
+ * @param startTime
```

Comment:
Params need descriptions. Also as it is now, you could make `startTime` default to `0` (since that's the only value you actually pass in).